setup.py

```diff
@@ -0,0 +1,32 @@
+from setuptools import setup
+
+import os
+
+package_files = []
+for dir, subdirs, files in os.walk("schema"):
+    for d in subdirs:
+        package_files.append(dir + "/" + d)
```

Comment:
It seems like this should be doing `os.path.join(dir, d)` and not assuming `/`

Reply:
`setuptools` actually wants _only_ slash-delimited paths here: 
> Also notice that if you use paths, you must use a forward slash (/) as the path separator, even if you are on Windows. Setuptools automatically converts slashes to appropriate platform-specific separators at build time.

(from deep in the bowels of https://setuptools.readthedocs.io/en/latest/setuptools.html#including-data-files)

Reply:
Some things are too clever. Of course using `posixpath.join` will ensue that 😉 

Also I tend to prefer f-strings (`f"{dir}/{d}"`)...